documentcloud/addons/models.py

```diff
@@ -577,6 +582,44 @@ def dispatch(self, document_pk=None):
             )
 
 
+class AddOnDisableLog(models.Model):
+    """Log usage of Add-On Disables"""
+
+    # Reference to the AddOnEvent
+    addon_event = models.ForeignKey(
+        verbose_name=_("event reference"),
+        to="addons.AddOnEvent",
+        on_delete=models.PROTECT,
+        related_name="disable_logs",
+        help_text=_("Reference to the Add-On Event"),
+    )
+
+    event = models.IntegerField(
```

Comment:
Why store event if it is always disabled?

Reply:
I was thinking the same, as stored AddOnEvent anyway. Leftover from last push. Will remove